README.md

```diff
@@ -7,12 +7,6 @@ _Our mission is to create an open platform for smart and ubiquitous computing, w
 
 A hosted version of Arcs is available at https://live.arcs.dev.
 
-> Note: Arcs is transitioning to Cloud Build and the live.arcs.dev hosted pages
```

Comment:
How do we know of errors in depolyment? Is there a badge we can add to the home page?

Reply:
Yes, and I believe we can get the URL for the badge from the `actions` tab. Let me see if I can get it before we start running the pipelines...

Reply:
It looks like the action has to land before I can generate a badge.